frontend/pages/hosts/components/DeleteHostModal/DeleteHostModal.tsx

```diff
@@ -31,24 +37,31 @@ const DeleteHostModal = ({
         <p>
           This action will delete{" "}
           <b>
-            {selectedHostIds.length}
+            {hostName}
+            {selectedHostIds?.length}
             {isAllMatchingHostsSelected && "+"}{" "}
-            {selectedHostIds.length === 1 ? "host" : "hosts"}
+            {selectedHostIds?.length === 1 ? "host" : "hosts"}
```

Comment:
when `selectedHostIds?.` evaluates to undefined the condition is `undefined === 1` which will render "hosts". If `selectedHostIds` can be undefined we need to handle the case when it is, which I think is we wouldn't render some of this text at all, or render some other tests.

if its the case that `selectHostIds` will never be undefined we need to reflect that in our typing.

Reply:
Ok, I updated it so hopefully it's more clear.

selectedHostIds can be undefined (if you click on the delete button from the host details page) in which it renders host name. I re-wrote the logic as a function so hopefully it's more readable that way. Let me know what you think! @ghernandez345